data/src/split.py

```diff
@@ -55,4 +56,15 @@ def main() -> None:
 
 
 if __name__ == "__main__":
-    main()
+    parser = argparse.ArgumentParser(
+        description="Sets path to tsv",
+    )
+    parser.add_argument(
+        "-tsv_path",
+        type=str,
+        metavar="",
+        required=True,
+        help="Declare path to tsv files",
```

Comment:
This is somewhat surprising but the autogenerated help strings begin wtih lowercase so the ones I write do too. And TSV is an acronym so it should be capitalized in documentation: "path to TSV files" would then suffice.